client/app/application.coffee

```diff
@@ -87,9 +78,21 @@ class App extends Application
 
 
     # Handler for register route, display onboarding's current step
-    handleRegisterRoute: =>
+    initializeOnboarding: =>
```

Comment:
As this method will be called every time the route will change, i.e. every time a step is validated and the next one "loaded", I think it is not relevant to name it initializeOnboarding, because we do not initalize the onboarding object at every step. We have to do it only one time at the first load.